README.md

```diff
@@ -123,22 +123,118 @@ const jsonTree = new StaticSiteJson('really-strange_placeToPut_some_FILES', {
 
 **Note:** just like the folder example the type will be automatically pluralised.
 
-### Collections
+### Collate
+If you want to have the ability to query all of your content at once you can do
+that by **collating** content together in a collection. This will place all of
+your markdown files into a single JSON:API document and can be used for
+`findAll` queries. To turn on collation you just need to set the `collate`
+attribute to `true`
 
-Collections are a convenient way of placing multiple markdown files (found under the same folder) in
-a single JSON:API document. This can be used when wanting to retrieve multiple documents at any one
-time (`findAll`).
+```javascript
+new StaticSiteJson(`content`, {
+  collate: true,
+})
+```
+
+* `collate`: Boolean - Default: false
+
+### CollationFileName
+
+If you have turned on collation by default BroccoliStaticSiteJson will  output
+the collated documents with the file name `all.json`. If you want to be able to
+edit this default output file you can set the `collationFileName`.
+
+```javascript
+new StaticSiteJson(`content`, {
+  collate: true,
+  collationFileName: 'articles.json'
+})
+```
+
+* `collationFileName`: String - Default: `all.json`
+
+### paginate
+
+In most cases when you're using BroccoliStaticSiteJson you probably will not be
+dealing with collections that are too large. But in some cases, for example a
+blog, you want to be able to deal with collections of an arbitrary length and it
+would be useful to paginate your collated collections. To enable pagination you
+set `paginate` to be true in your options:
 
 ```javascript
 new StaticSiteJson(`content`, {
-  collections: [{
-    output: `allContent.json`,
-  }]
+  collate: true,
+  paginate: true,
 })
 ```
 
-* `options`
-  * `output`: The output file name of the collection JSON:API document.
+**Note:** `paginate` will do nothing if you haven't set `collate` to true.
```

Comment:
Should we default to setting `collate` to true, when `paginate` is true?

Reply:
to be honest it's just the way the `if()` statements work in the code 🤔 I think it's reasonable to let it be explicit like this.

If we were going to automatically set `collate` to true then, to be consistent` we would need to set `paginate` to true when `pageSize` is set to true 😂 I think being consistent on the explicit side is better overall 👍